ee/dtr/admin/configure/deploy-caches/simple-kube.md

```diff
@@ -0,0 +1,229 @@
+---
+title: Deploy a DTR cache with Kubernetes
+description: Deploy a DTR cache to make users in remove geographical locations
+  pull images faster.
+keywords: DTR, cache, kubernetes
+---
+
+This example guides you through deploying a DTR cache, assuming that you've got
+a DTR deployment up and running. It also assumes that you've provisioned
+a Kubernetes Cluster of 1.8 or higher.
+
+The DTR cache is going to be deployed as a Kubernetes Deployment, so that Kubernetes
+automatically takes care of scheduling and restarting the service if
+something goes wrong.
+
+We'll manage the cache configuration using a Kubernetes Config Map, and the TLS
+certificates using Kubernetes secrets. This allows you to manage the configurations
+securely and independently of the node where the cache is actually running.
+
+## Prepare the cache deployment
+
+At the end of this exercise you should have a file structure that looks like this:
+
+```
+├── dtrcache.yml        # Yaml file to deploy cache with a single command
+├── config.yml          # The cache configuration file
+└── certs
+    ├── cache.cert.pem  # The cache public key certificate
+    ├── cache.key.pem   # The cache private key
+    └── dtr.cert.pem    # DTR CA certificate
+```
+
+### Create the DTR Cache certicates
+
+The DTR cache will be deployed with a TLS endpoint. For this you will need to generate
+a SSL ceritificate and key from a certificate authority. Depending on how you would
+ like to expose the DTR Cache it will depend on the SANs required for this certificate.
+
+For example:
+
+  - If you are deploying the DTR Cache with an 
```

Comment:
What is the FQDN for an ingress object given how our "https://docs.docker.com/ee/ucp/kubernetes/layer-7-routing/" doesn't really work with UCP 3.1?